Testing/functional/e2e/cypress/integration/dependent/dependents.js

```diff
@@ -129,7 +129,7 @@ describe('dependents', () => {
     it('Validate Covid Tab with Results', () => {
         // Validate the tab and elements are present        
         cy.get('[data-testid=covid19TabTitle]').last().parent().click();
-        cy.get('[data-testid=dependentCovidTestDate]').first().should('have.text', ' 2020-08-21 ');
+        cy.get('[data-testid=dependentCovidTestDate]').first().should('have.text', ' 2020-10-03 ');
```

Comment:
Should we add some steps to validate the sort is correct for these items?

Reply:
I dont understand, it already does that, it checks for the first and the last element dates, whats your suggestion?

Reply:
ah right, I didn't check the last item below